spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -670,20 +716,30 @@ The IBC handler performs the following steps in order:
 ```typescript
 function writeAcknowledgement(
   packet: Packet,
-  acknowledgement: bytes): Packet {
+  acknowledgement: Acknowledgement | bytes): Packet {
```

Comment:
Not sure how to write both versions here....

This is a "v2" handler that may be talking on a  "v1" or "v2" connection